Yoast/javascript
# Update babel-preset-es2015 to babel-preset-env

## Summary

In my-yoast, we need to remove babel-preset-es2015 because it is deprecated https://github.com/Yoast/my-yoast/issues/2211.

In the process of removing it https://github.com/Yoast/my-yoast/pull/2638, we discovered we couldn't remove babel-preset-2015 because yoast-components still used it. In This PR, I try to remove babel-presset-es2015 from yoast-components, so that it can be removed from my-yoast afterwards.


  * Package(s) involved: Yoast-components

  * Should the change be included in the package changelog?
    * [x] Yes. At least, I would expect so.
  * Should the change be included in one or more plugin changelogs?
  * Package changelog item (if applicable): Replaced babel-preset-es2015 with babel-preset-env, as babel-preset-es2015 is deprecated.
  * Plugin changelog item (if applicable): 

## Relevant technical choices:

* I took the presets from my-yoast for now. Not sure if that was the right decision.
* Tests still succeeded when I expected them to fail locally. I wouldn't be surprised if unit-tests fail for travis, as i've encountered this before, and am not sure what to do about it yet.

## Test instructions

This PR can be tested by following these steps: (targeted at a team-platform member, who might not have a proper yoast-seo test environment yet)

* Make sure you have your wordpress-seo development set-up working:
   * Have your VVV set up properly.
   * Run vagrant up in your VVV.
   * Make sure you have the WordPress-seo repository up-to-date and and ready:
      * Git pull;
      * composer install;
      * yarn install;
      * grunt build.
* Set up a local javascript repository;
   * For now, have develop branch checked out;
   * run yarn install in both the roof folder and in the packages/yoast-components folder.
* Make sure you can activate the wordpress-seo plugin on your local test environment.
* Link javascript to wordpress-seo by running `yarn link-monorepo` in the wordpress-seo repository;
* You can do a quick check whether the link succeeded by changing `isExpanded` to true in the file `yoast-components/composites/plugin/helpCenter/Helpcenter.js`. And then running grunt build:js in wordpress-seo. If you visit a yoast-seo settings page and the help center is opened on pageload, the link works as intended.
* Take note of any existing console errors in the wordpress-seo. Check a settings page and a post page at the very least. I ask this to exclude errors not caused by my branch. 
* Checkout this branch in javascript;
* run `yarn install` in both root and `packages/yoastcomponents` of the javascript repo;
* run grunt build:js in wordpress-seo;
* Make sure no new errors are introduced in the plugin;
* Make sure unit-tests pass for both wordpress-seo and javascript. Use `yarn test` for this. For the javascript repository, run `yarn test` in the `packages/yoast-components` directory, as otherwise, an awful lot of tests are run 😬 

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #288
